src/opt.rs

```diff
@@ -92,7 +92,7 @@ pub struct Opt {
     #[structopt(long = "min-cache", default_value = "2048")]
     pub min_cache: u64,
 
-    /// The number of CPU cores to use
+    /// The number of CPU cores to use. Ignored for workers.
```

Comment:
Maybe it's better to add a runtime check? This way we kinda enforce that this option does not work when in worker mode.